src/ui/libs/DatalensChartkit/ChartKit/plugins/Table/renderer/components/Table/cell-width.ts

```diff
@@ -0,0 +1,25 @@
+export function getCellsWidth(args: {
+    cols: {min: number; fixed?: number}[];
+    tableMinWidth: number;
+}): number[] {
+    const {tableMinWidth, cols} = args;
+    const shouldIgnoreFixedWidth = cols.every((col) => col.fixed);
```

Comment:
You can calculate this value outside this function and use its value as an argument

Reply:
I can, but why? It is only needed to calculate the width - it cannot be reused anywhere else